cmake/generic.cmake

```diff
@@ -597,6 +597,9 @@ function(grpc_library TARGET_NAME)
           COMMAND ${PROTOBUF_PROTOC_EXECUTABLE}
           ARGS --grpc_out "${CMAKE_CURRENT_BINARY_DIR}" -I "${PROTO_PATH}"
           --plugin=protoc-gen-grpc="${GRPC_CPP_PLUGIN}" "${ABS_PROTO}"
+          COMMAND ${PROTOBUF_PROTOC_EXECUTABLE}
```

Comment:
line 590 already calls `protobuf_generate_cpp(grpc_proto_srcs grpc_proto_hdrs "${ABS_PROTO}")`, curious why that doesn't work, and shoud line 590 be removed?

Reply:
hmm, i didn't notice this line, but this looks weird to me too. let me test if it's ok to remove the line 590.
also, I think it's good practice to put the generation command together, what's your idea?

Reply:
the build failed due to the missing of a dependency

```
The dependency target "sendrecvop_grpc_proto" of target "sendrecvop_grpc"
```

looks we still need to add line 590 back for now

Reply:
If this just works, plz add a note here and merge this, we really need this unittest.